go-controller/hybrid-overlay/pkg/controller/overlay_windows.go

```diff
@@ -256,6 +258,11 @@ func GetExistingNetwork(networkName string, expectedAddressPrefix string, expect
 		}
 	}
 
+	if existingNetwork != nil {
+		// the named network already exists but the nodes sunbet or GW has changed
+		existinNetwork.Delete()
```

Comment:
`existingNetwork` (was missing a g)

Note that a normal `make` won't build the windows stuff; you want `make windows` to catch build errors.

Reply:
sorry, stupid mistake. Corrected

Reply:
Is it possible to add the Windows build as a PR test?